config/settings_data.json

```
@@ -1 +1,63 @@
-{}
+{
+  "current": "Craft",
```

Shouldn't this be Default?

```suggestion
  "current": "Default",
```

---

🤦